core/field_colour.js

```diff
@@ -78,15 +78,15 @@ Blockly.FieldColour.fromJson = function(options) {
  * @private
  * @const
  */
-Blockly.FieldColour.DEFAULT_WIDTH = 16;
+Blockly.FieldColour.DEFAULT_WIDTH = 16 + Blockly.Field.X_PADDING;
```

Comment:
Since there's no padding on a colour field, I think it would make more sense to set `DEFAULT_WIDTH = 26`.

Reply:
Reasonable.  Done.